readthedocs/builds/syncers.py

```diff
@@ -53,30 +54,26 @@ def copy(cls, path, target, is_file=False, **__):
         sync_user = getattr(settings, 'SYNC_USER', getpass.getuser())
         app_servers = getattr(settings, 'MULTIPLE_APP_SERVERS', [])
         if app_servers:
-            log.info("Remote Copy %s to %s on %s", path, target, app_servers)
+            log.info('Remote Copy %s to %s on %s', path, target, app_servers)
             for server in app_servers:
-                mkdir_cmd = ("ssh %s@%s mkdir -p %s" % (sync_user, server, target))
+                mkdir_cmd = (
+                    'ssh %s@%s mkdir -p %s' % (sync_user, server, target))
                 ret = os.system(mkdir_cmd)
                 if ret != 0:
-                    log.info("COPY ERROR to app servers:")
-                    log.info(mkdir_cmd)
+                    log.error('Copy error to app servers: cmd=%s', mkdir_cmd)
                 if is_file:
-                    slash = ""
+                    slash = ''
                 else:
-                    slash = "/"
+                    slash = '/'
                 # Add a slash when copying directories
                 sync_cmd = (
                     "rsync -e 'ssh -T' -av --delete {path}{slash} {user}@{server}:{target}"
                     .format(
-                        path=path,
-                        slash=slash,
-                        user=sync_user,
-                        server=server,
+                        path=path, slash=slash, user=sync_user, server=server,
```

Comment:
This squashed the continuation

Reply:
I had sent a comment here, but my internet connection didn't help me:

----

By adding a `,` after the `target` here, we get a one attribute per line formatting automatically by yapf.

There are many changes on these files were we can apply this fix to get the more readable way, but I don't want to do it manually for now.

I think we can easily add the trailing comman once we start using pre-commit in a daily basis.

----

Anyway, I think we can use this PR to discuss/tweak yapf a little more.

There are things that need manual work. Although, for this particular issue that the trailling comma is missing we could add automatically before calling yapf with this hook: https://github.com/asottile/add-trailing-comma